robottelo/cli/contentview.py

```diff
@@ -208,3 +208,9 @@ def component_list(cls, options=None):
         """List components attached to the content view"""
         cls.command_sub = 'component list'
         return cls.execute(cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def list(cls, options=None):
+        """List components attached to the content view"""
```

Comment:
Just fix the description... Probably "List content views"

Reply:
Yeah, this was a copy paste situation 😂